frontend/src/pages/AboutUs/AboutUsPage.module.scss

```diff
@@ -169,7 +168,7 @@
 
 // Grid rows with outlines.
 .GridOutlinedRow {
-  padding: 147px 120px 147px 120px;
+  padding: 147px 120px 0px 120px;
```

Comment:
This change makes sense to me everywhere else, it just makes the "Where we Started" row on the "The Project" tab look uncentered.  Can you replace the original padding / implement a different solution rq?